services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/resilience/StormLcmSpec.groovy

```diff
@@ -52,16 +52,19 @@ class StormLcmSpec extends HealthCheckSpecification {
         }
 
         and: "Database dump"
-        def nodesDump = database.dumpAllNodes()
         def relationsDump = database.dumpAllRelations()
+        def switchesDump = database.dumpAllSwitches()
+        def islsDump = database.dumpAllIsls()
```

Comment:
islsDump is a subset of relationsDump. Makes no sense since you already do relationsDump

Reply:
islDump wass removed from test